docs/asciidoc/http-client.adoc

```diff
@@ -512,3 +518,188 @@ include::{examplesdir}/resolver/custom/Application.java[lines=18..38]
+* <<http-client-timeout>>
+** <<response-timeout>>
+** <<connection-timeout>>
+** <<ssl-tls-timeout>>
+** <<proxy-timeout>>
+** <<dns-timeout>>
+
+[[connection-pool-timeout]]
+=== Connection Pool Timeout
+By default, `HttpClient` uses a connection pool. When a request is completed successfully and if the connection is not scheduled for closing,
+the connection is returned to the connection pool and thus it can be reused for processing another request. The connection may
+be reused immediately for another request or may stay idle in the connection pool for some time.
+
+The below shows the available timeout configuration options:
+
+* `maxIdleTime` - the maximum time (resolution: ms) that this connection stays idle in the connection pool. By default,
+{javadoc}/reactor/netty/resources/ConnectionProvider.ConnectionPoolSpec.html#maxIdleTime-java.time.Duration-[maxIdleTime] is not specified.
+
+NOTE: When you configure `maxIdleTime`, you should consider the idle timeout configuration on the target server.
+Choose a configuration equal or less than the one on the target server thus you will be able to reduce the I/O
+issues caused by a connection closed by the target server.
+
+* `maxLifeTime` - the maximum time (resolution: ms) that this connection stays alive. By default,
+{javadoc}/reactor/netty/resources/ConnectionProvider.ConnectionPoolSpec.html#maxLifeTime-java.time.Duration-[maxLifeTime] is not specified.
+* `pendingAcquireTimeout` - the maximum time (resolution: ms) after which a pending acquire operation must complete, or
+a `PoolAcquireTimeoutException` will be thrown. Default: 45s.
+
+By default, these timeouts are checked on connection `release`/`acquire` operation and if some timeout is reached, the connection is closed and removed from the connection pool.
+However, you can also configure the connection pool, via `evictInBackground`, to perform periodic checks on connections.
+
+To customize the default settings, you can configure `HttpClient` as follows:
+
+====
+[source,java,indent=0]
+.{examplesdir}/pool/Application.java
+----
+include::{examplesdir}/pool/Application.java[lines=18..50]
+----
+<1> Configures the maximum time for a connection to stay idle to 20 seconds.
+<2> Configures the maximum time for a connection to stay alive to 60 seconds.
+<3> Configures the maximum time for the pending acquire operation to 60 seconds.
+<4> Every 2 minutes the connection pool is regularly checked for connections, that are applicable for removal.
+====
+
+[[http-client-timeout]]
+=== HttpClient Timeout
+This section provides information for the various timeout configuration options at the `HttpClient` level.
+
+NOTE: Reactor Netty uses Reactor Core as Reactive Streams implementation, and you may want to use the `timeout` operator that `Mono`/`Flux` provide.
+Keep in mind however that it is better to use more specific timeout configuration options available in Reactor Netty since they provide more control for a specific purpose and use case.
```

Comment:
Change to:

Keep in mind, however, that it is better to use the more specific timeout configuration options available in Reactor Netty, since they provide more control for a specific purpose and use case.